src/containers/Charts/Chart.css

```diff
@@ -37,7 +37,15 @@
     background-color: whitesmoke;
     padding: 1em;
     margin: 1em;
-    height: 35em;
-    width: 40em;
+    height: 40vh;
+    width: 40vw;
```

Comment:
Per the chart.js docs, the height and width need to be 'vh/vw' in order for the graphs to be responsive.